services/archlinux/archlinux.tester.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const { isVPlusDottedVersionNClauses } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('Arch Linux package (valid)')
+  .get('/core/x86_64/pacman.json')
+  .expectBadge({ label: 'arch linux', message: isVPlusDottedVersionNClauses })
```

Comment:
Similar question here as on the Debian PR. Is a `pacman` version always going to match this regex, or should we be a little more lenient?

Reply:
Same as in Debian, done here too.